### PR TITLE
Package.json and changelog

### PR DESCRIPTION
**Pull request by nturinski**

### CODE REVIEW

**Review by alexweininger — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # Change Log
 
+## 1.14.2 - 2024-04-26
```
Comment (motm32) [inline CHANGELOG.md:3]: Missing a line after the date.

Comment (nturinski) [reply CHANGELOG.md:3]:
```suggestion
## 1.14.2 - 2024-04-26

```

Comment (nturinski) [reply CHANGELOG.md:3]: You have just witnessed a master in laziness 🙇

Comment (alexweininger) [reply CHANGELOG.md:3]: omg

Comment (alexweininger) [reply CHANGELOG.md:3]: also ftr that newline doesn't actually impact the rendered markdown but it is good to be consistent

**Review by motm32 — APPROVED** [review]

**Pull request merged**